pynest/nest/tests/test_fixed_multi_cmpt_neu.py

```diff
@@ -0,0 +1,359 @@
+		self.refca = 999
+		self.catmp = 0.0
+		self.reftmp = 0
+
+
+		self.r = ode(self.fun_u).set_integrator('dop853',rtol=1e-3, nsteps=500)
+		self.r.set_initial_value(self.u,0.)
+	
+	def compute_u(self,G_exc,G_inh,I_curr):
+
+
+
+		self.counter = 0
+		self.distal_cnt = 0
+
+		self.r.y[11] += (G_exc[0,0]*np.exp(1)/self.Tau_synE[0,0])
+		self.r.y[13] += (G_inh[0,0]*np.exp(1)/self.Tau_synI[0,0])
+		self.r.y[7] += (G_exc[1,0]*np.exp(1)/self.Tau_synE[1,0])
+		self.r.y[9] += (G_inh[1,0]*np.exp(1)/self.Tau_synI[1,0])
+		self.r.y[3] += (G_exc[2,0]*np.exp(1)/self.Tau_synE[2,0])
+		self.r.y[5] += (G_inh[2,0]*np.exp(1)/self.Tau_synI[2,0])
+
+		self.r.integrate(self.r.t+self.resolution)
+		self.u = self.r.y
+
+		if self.reftmp > 0:
+			self.reftmp -=1
+			self.catmp = self.ca[self.refca - self.reftmp]
+		else:
+			if self.u[2] >= self.thca:
+				self.reftmp = self.refca
+				self.catmp = self.ca[0]
+		
+
+class FixedMultiCmptNeuTestCase(unittest.TestCase):		
```

Comment:
This `TestCase` also needs the decorators:

    @nest.check_stack
    @unittest.skipIf(not HAVE_GSL, 'GSL is not available')